plugin/src/ios/CDVRadar.m

```diff
@@ -158,8 +219,25 @@ - (void)requestPermissions:(CDVInvokedUrlCommand *)command {
 }
 
 - (void)getLocation:(CDVInvokedUrlCommand *)command {
+    NSString *desiredAccuracy = [command.arguments objectAtIndex:0];
```

Comment:
does this line work when `location` is not passed in?  Can you add a call to the example app?

Reply:
sure

Reply:
Sorry, I mean when `desiredAccuracy` is not passed in.